### DISCUSSION TITLE
Steps to run cloudflared in a non-supervisor environment?

### DISCUSSION THREAD

**Discussion by gcormier** [homeassistant-apps/app-cloudflared#547] [Ideas]
I've started adding cloudflared to my docker-compose, but there's not enough documentation to really proceed (eg mount points) and even know if it will work.

Since HA container install is a supported config, any chance cloudflared can work?

**Comment by elcajon** [comment]
Just install Cloudflared following their official guides. This add-on just works with supervised/os installations.
